sql-parser/src/main/java/io/crate/sql/SqlFormatter.java

```diff
@@ -738,6 +737,63 @@ public Void visitCreateSnapshot(CreateSnapshot node, Integer indent) {
             return null;
         }
 
+        @Override
+        public Void visitDropTable(DropTable node, Integer indent) {
+            builder.append("DROP TABLE ");
+            if (node.dropIfExists()) {
+                builder.append("IF EXISTS ");
+            }
+            process(node.table(), indent);
+            return null;
```

Comment:
This looks a bit odd to me, but maybe i don't get the big picture. 
What is the purpose of declaring the return type as java.lang.Void and then return null ? Can't we just go with a void method ?

Reply:
It's a visitor implementation of Visitor<ReturnType, ContextType> so the return type must be defined.

Reply:
ok, understood, makes sense. Thanks for the clarification.